en/guides/codablerouting.md

```diff
@@ -15,172 +15,263 @@ redirect_from:
+### Getting started
+First we need a Kitura server to add our routes to. Kitura has a [command line interface](https://github.com/IBM-Swift/kitura-cli) which provides us with utilities for quickly creating a Kitura server.  
+> ![tip] You can find installation instructions for the command line interface in the [Getting Started](/{{ page.lang }}/starter/gettingstarted.html) section. 
 
-and then provide an implementation of the storeHandler:
+Once we've installed the Kitura command line interface we can open a terminal window and run the following:  
 
-```swift
-func storeHandler(todo: ToDo, completion: (ToDo?, RequestError?) -> Void ) -> Void {
-    todoStore.append(todo)
-    completion(todo, nil)
-}
-```
+1. Create a directory:  
+`mkdir ~/CodableRouting`
+2. Change to the new directory  
+`cd ~/CodableRouting`
+3. Create the Kitura server  
+`kitura init`
+
+`kitura init` downloads, creates and builds a starter Kitura application and will take a couple of minutes to complete. This includes best-practice implementations of health checking and monitoring as well as configuration files to allow easily deployment to a Docker container, a Kubernetes cluster, or the IBM Cloud.
 
-Here the store handler accepts a todo item of type `ToDo` and responds using an asynchronous completion handler. The handler then stores the received todo item in the todoStore and returns with the created todo item in the case of success, and an `Error` in the the case of failure.
+`kitura init` also creates an Xcode project for our server so we can open that now:  
+`open CodableRouting.xcodeproj`
 
-The input parameter and the response parameter in completion handler can be any Swift type you want, as long as it conforms to Codable. For example, we could use the following as our `ToDo` type:
+Now we're ready to start adding our Codable routes!
 
+### Creating the models
+Codable routing works directly with Swift value types. So we need to contruct a value type to use. A good example of this is a struct that conforms to Codable: 
 ```swift
-public struct ToDo : Codable {
-    public var title: String        // title of the todo item
-    public var order: Int           // order to display the todo item
-    public var completed: Bool      // has the item been completed
+struct User: Codable {
+    let name: String
+    let age: Int
 }
 ```
 
-At this point the server has stored the item, but the client has no way of referring to that specific item on the server if it wants to get, update, or delete that item. In order to do that, some kind of unique identifier is required.
+It's good practice to keep value types (in this case models) and logic separated, we will create a new file called `Models.swift` as follows:  
+1. In Xcode right click on the `Application` folder and select `New File`
+2. Select `Swift File` in the UI pop up and click next
+3. In the `Save As` field, name the file `Models.swift`
+4. Ensure Group is `Application` and target is `Application` as well
```

Comment:
lose the words "as well"

Please can you put full stops at the end of the sentences in 1-5